.travis.yml

```diff
@@ -9,7 +9,7 @@ scala:
 
 script:
 - sbt ++$TRAVIS_SCALA_VERSION orgScriptCI
-- sbt ++$TRAVIS_SCALA_VERSION scripted
+- sbt ++$TRAVIS_SCALA_VERSION publishLocal scripted
```

Comment:
Why adding `publishLocal` here?

Reply:
By default, scripted doesn't make the publishLocal for the dependent libraries (client)